flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -44,6 +44,15 @@
                     .noDefaultValue()
                     .withDescription("The state backend to be used to checkpoint state.");
 
+    /** Whether to enable state change log. */
+    // @Documentation.Section(value = Documentation.Sections.COMMON_STATE_BACKENDS)
+    public static final ConfigOption<Boolean> ENABLE_STATE_CHANGE_LOG =
+            ConfigOptions.key("state.backend.enable-statechangelog")
```

Comment:
I think this option should be hidden.

Reply:
do you mean we should hide it from users for now or we should not use this option?

Reply:
I mean to hide it from users (with `Documentation.ExcludeFromDocumentation`).